templates/index.html

```diff
@@ -70,34 +74,75 @@ <h1 class="row text-center">Get Sponge</h1>
     </div>
 </section>
 
-<!-- Contact Section -->
-<section class="sponsors" id="sponsors">
+<!-- Contribute Section -->
+<section class="contribute" id="contribute">
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h1>Sponsors</h1>
+                <h1>Contribute</h1>
                 <br>
-                <h3>Sponge is proudly sponsored by</h3>
+                <h3>There are a number of ways you can contribute to Sponge:</h3>
                 <br>
             </div>
         </div>
         <div class="row">
-            <div class="col-md-12 text-center">
-                {% for sponsor in sponsors %}
-                <div class="{{ sponsor.ColumnSize }} portfolio-item">
-                    <a href="{{ sponsor.Link }}" target="_blank" class="nodeco sponsor">
-                        <img src="{{ sponsor.Image }}" width="{{ sponsor.Size }}" height="{{ sponsor.Size }}"
-                             class="gray" alt="{{ sponsor.Name }}">
-                    </a>
-                </div>
-                {% endfor %}
+            <div class="col-lg-4 text-center">
+                <h1>Code</h1>
+                <br>
+                <p>
+                    Are you a Java programmer? Want to contribute to the most ground-breaking project the Minecraft
+                    community has ever seen, or just love testing stuff to destruction and then writing long and
+                    detailed bug reports about it? Come get involved on GitHub!
+                </p>
+            </div>
+            <div class="col-lg-4 text-center">
+                <h1>Docs</h1>
+                <br>
+                <p>
+                    Coding isn't your thing but you love helping others, get a thrill from hammering out the perfect
+                    prose, or just love nit-picking other peoples' grammar? Our fearsome and frankly unstoppable
+                    documentation team will be delighted to induct you into their team of infamous brigands and
```

Comment:
team is used twice in the sentence i'd change the second reference to something like ranks